nemo/collections/common/data/prompt_fn.py

```diff
@@ -0,0 +1,78 @@
+from typing import Callable, Type
+
+import torch
+
```

Comment:
This file seems to have a lot of overlap with `https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/common/prompts/fn.py`?

Reply:
Good catch, `prompts/fn.py` was supposed to be removed. Updated.